diff -Naur ./CMakeLists.txt ./CMakeLists.txt
--- ./CMakeLists.txt	2014-04-09 15:09:10.000000000 -0600
+++ ./CMakeLists.txt	2016-05-01 02:49:55.361105001 -0600
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8)
 project( fontconfig )
-set( CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules )
+# set( CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules ) # commented out by SuperBuild to use SuperBuild find package scripts
 
 find_package( EXPAT REQUIRED )
 find_package( Freetype REQUIRED )
diff -Naur ./conf.d/CMakeLists.txt ./conf.d/CMakeLists.txt
--- ./conf.d/CMakeLists.txt	2014-04-09 15:09:10.000000000 -0600
+++ ./conf.d/CMakeLists.txt	2016-05-01 02:48:15.951424999 -0600
@@ -1,3 +1,3 @@
 file( GLOB conf_files RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} "*.conf" )
-install( FILES README ${conf_files} DESTINATION etc/fonts/conf.d )
-install( FILES README ${conf_files} DESTINATION etc/fonts/conf.avail )
\ No newline at end of file
+install( FILES README.in ${conf_files} DESTINATION etc/fonts/conf.d )
+install( FILES README.in ${conf_files} DESTINATION etc/fonts/conf.avail )
diff -Naur ./src/CMakeLists.txt ./src/CMakeLists.txt
--- ./src/CMakeLists.txt	2014-04-09 15:09:10.000000000 -0600
+++ ./src/CMakeLists.txt	2016-05-01 02:48:57.916397000 -0600
@@ -28,6 +28,7 @@
     fcstr.c
     fcxml.c
     ftglue.c
+    fcrange.c
 )
 
 include_directories(
